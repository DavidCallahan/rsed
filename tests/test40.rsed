input split "a b  c  d e" with " +"
copy all
close input
input split "012345678901234" columns 3 5 7
copy all
