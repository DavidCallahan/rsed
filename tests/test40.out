a
b
c
d
e
012
34
56
78901234
